nebula/ui/components/VPNSegmentedToggle.qml

```diff
@@ -75,7 +75,19 @@ Rectangle {
 
                 objectName: segmentButtonId
 
+
+                Accessible.role: Accessible.Button
+                Accessible.description: `${index + 1} of ${options.count}`
+                Accessible.name: {
+                    var name = ""
+                    if(root.selectedIndex === index) name += "selected,"
+                    name += label.text
+                    return name
+                }
+                Accessible.onPressAction: segmentClicked()
```

Comment:
Screen reader reads out the following: 

- Selected segment: "Selected, \<button text>, button, \<x> of \<y>" 
- Unselected segment: "\<button text>, button, \<x> of \<y>"

where `x` is the index of the segment, and `y` is the total number of segment options in the model. This reads out nearly the same way a native iOS UISegmentedControl reads (pausing is a little different).

Open to ideas if there is a better way to do this!

Reply:
Makes sense to me. The only other thing coming to mind is maybe changing the delegates to Buttons and adding them to a ButtonGroup.

Reply:
I think a ButtonGroup may be redundant here since the control already handles selecting a segment and preventing segments from being selected again while they are already selected.

Will definitely change the delegate to buttons though, since thats what they essentially are!